configure-ironic.sh

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/bash
 
+if [ -n "$IRONIC_CERT_FILE" ] && [ -z "$IRONIC_KEY_FILE" ] ; then
+    echo "Missing TLS Certificate key file"
+    exit 1
+fi
+if [ -z "$IRONIC_CERT_FILE" ] && [ -n "$IRONIC_KEY_FILE" ] ; then
+    echo "Missing TLS Certificate file"
+    exit 1
+fi
```

Comment:
Maybe this is just me, but I don't see the need to do two lots of configuration - mounting the files somewhere and then telling the container where you've mounted them. You could just have a well-defined place where this script looks for the files, and you either mount them there or you don't. That eliminates the possibility of a mismatch, and the need to handle that case (which currently we just leave up to Ironic, by the looks of it).

Reply:
ok, thanks for the idea, I will do that.